testplan/vendor/tempita/__init__.py

```diff
@@ -1172,6 +1172,7 @@ def get_token(pos=False):
             start_pos = end_pos = None
             parts = []
             while 1:
+                # pylint: disable=unbalanced-tuple-unpacking
                 tok_type, tok_string, s, e = get_token(True)
```

Comment:
I know you didn't add this but we're confident that we'll never hit [this StopIteration](https://github.com/Morgan-Stanley/testplan/blob/668ca2d1d9ed25e9679f25952e55fce77b1a7399/testplan/vendor/tempita/__init__.py#L1134) exception? It seems like [this generator](https://github.com/Morgan-Stanley/testplan/blob/668ca2d1d9ed25e9679f25952e55fce77b1a7399/testplan/vendor/tempita/__init__.py#L1124) will run out eventually, thus throwing StopIteration, thus returning only two elements, thus throwing an exception here since we're expecting at least 4 elements.